packages/cli/odahuflow/cli/utils/output.py

```diff
@@ -52,7 +52,7 @@
 def validate_output_format(ctx, param, value):
     """
     Custom validator for "--output-format" parameter.
-    It's an analog of click.Choise
+    It's an analog of click.Choice
```

Comment:
```suggestion
    It's an analogue of click.Choice
```

Reply:
analog US/ analogue UK. So I just leave as it is.